modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase

```diff
@@ -246,6 +246,108 @@ definition {
 			locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
 	}
 
+	@description = "This is a test for COMMERCE-11957. As a buyer, I want to filter products using Specification Facet widget by specification value and the results are visible on the Search Results widget"
+	@priority = 5
+	test AssertProductsAreFilteredBySpecificationValue {
```

Comment:
As the focus of the test is to not be able to have a dot in the specification key, please find a name more suitable for the covered scenario